src/spec/views/HomeContentView_spec.js

```diff
@@ -0,0 +1,51 @@
+import HomeContentView from '../../views/HomeContentView';
+import Mediator from '../../lib/Mediator.js';
+
+describe('mediated event handling', function () {
+
+    let homeContentView, mediator;
+
+    beforeEach(function () {
+        mediator = new Mediator();
+
+        homeContentView = new HomeContentView({templateId: 'NSIDC', mediator: mediator});
+        homeContentView.render();
+    });
+
+    it('is bound to the app:home event', function () {
+        jest.spyOn(homeContentView, 'onAppHome');
+
+        homeContentView.setMediator(mediator);
+
+        mediator.trigger('app:home');
+        expect(homeContentView.onAppHome).toHaveBeenCalled();
+    });
+
+    it('is hidden when a search is intiated', function () {
+        mediator.trigger('search:initiated');
+        expect(homeContentView.el).toHaveClass('hidden');
+    });
+
+    it('triggers a search when an example term is selected', function () {
```

Comment:
This is a good test -- however, I haven't been able to find evidence that the "example terms" concept still exists in the live interface. Do you know which UI elements are actually involved here?

Reply:
No, upon closer examination, it doesn't seem like it - there was a callback for it in SearchCriteriaView, but nothing in the app triggers that event anymore.  I removed this test and the related defunct code.